src/pybind/mgr/dashboard/frontend/src/app/ceph/smb/smb-cluster-form/smb-cluster-form.component.html

```diff
@@ -0,0 +1,343 @@
+<div cdsCol
+     [columnNumbers]="{ md: 4 }"
+     *ngIf="orchStatus$ | async as orchStatus">
+  <form name="smbForm"
+        #formDir="ngForm"
+        [formGroup]="smbForm"
+        novalidate>
+    <div i18n="form title"
+         class="form-header">
+      {{ action | titlecase }} {{ resource | upperFirst }}
+    </div>
+
+    <!-- Cluster Id -->
+    <div class="form-item">
+      <cds-text-label
+        for="cluster_id"
+        i18n
+        helperText="Unique cluster identifier"
+        i18n-helperText
+        cdRequiredField="Cluster Name"
+        [invalid]="smbForm.controls.cluster_id.invalid && smbForm.controls.cluster_id.dirty"
+        [invalidText]="clusterError"
+        >Cluster Name
+        <input
+          cdsText
+          type="text"
+          placeholder="Cluster Name..."
+          i18n-placeholder
+          id="cluster_id"
+          name="cluster_id"
+          formControlName="cluster_id"
+          [invalid]="smbForm.controls.cluster_id.invalid && smbForm.controls.cluster_id.dirty"
+        />
+      </cds-text-label>
+      <ng-template #clusterError>
+        <span
+          class="invalid-feedback"
+          *ngIf="smbForm.showError('cluster_id', formDir, 'required')"
+          i18n
+          >This field is required.</span
+        >
+      </ng-template>
+    </div>
+
+    <!-- Auth Mode -->
+    <div class="form-item">
+      <cds-select
+        formControlName="auth_mode"
+        name="auth_mode"
+        for="auth_mode"
+        label="Authentication Mode"
```

Comment:
```suggestion
        label="Authentication Mode"
```

id is used as for.
See 		https://github.com/carbon-design-system/carbon-components-angular/blob/v5.56.2/src/select/select.component.ts#L67